catalog/spatial/csw/spatial-csw-endpoint/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswQueryFactoryTest.java

```diff
@@ -1034,9 +1040,9 @@ private DistanceBufferType createDistanceBufferType() {
 
         QueryImpl frameworkQuery = (QueryImpl) queryFactory.getQuery(grr)
                 .getQuery();
-        assertThat(frameworkQuery.getFilter(), instanceOf(clz));
+        assertThat(((QueryImpl) frameworkQuery.getFilter()).getFilter(), instanceOf(clz));
```

Comment:
[pencil] This whole QueryImpl.getFilter() call with case is used again two lines later. Maybe it would be better off as a local variable.